### ISSUE TITLE
Handle $0 purchase prices

### ISSUE THREAD

**Issue by bberenberg** [Atlas-Authority/marketing-automation#109]
Looks like there are $0 purchase price transactions being backfilled into the historical transactions. We will need to handle this somehow. Not sure if we should create $0 deals or skip them or what.

**Comment by sdegutis** [comment]
I think this is done in #111 right?

**Comment by bberenberg** [comment]
No, #111 is the short term workaround so that we can keep working even though they released this to prod. It's us opting out of $0 values. We want to actually handle them.

**Comment by sdegutis** [comment]
Oh ok understood. It seems there's an open question as to how to handle $0 purchases and what specifically should be done. That's in business scope and not tech scope so I'm not sure I can weigh in much there. So this is blocked until that's answered.

**Comment by bberenberg** [comment]
Let's handle them as new deals with $0 value. I guess this will allow for more consistent reporting.
